features/authentication-framework/org.wso2.carbon.identity.application.authentication.framework.server.feature/resources/org.wso2.carbon.identity.application.authentication.framework.server.feature.default.json

```diff
@@ -146,6 +146,21 @@
   "authentication.authenticator.sms_otp.parameters.SendOTPDirectlyToMobile": false,
   "authentication.authenticator.sms_otp.parameters.redirectToMultiOptionPageOnFailure": false,
 
+  "authentication.authenticator.totp.name": "totp",
+  "authentication.authenticator.totp.enable": true,
+  "authentication.authenticator.totp.parameters.useEventHandlerBasedEmailSender": false,
```

Comment:
"useEventHandlerBasedEmailSender" is not a good term to be used with the new config model. This represents some internal detail which external users are unaware. From external user's POV, they need to enable HTML based emails, and the config name should be intuitive.

Reply:
created a key mapping

Reply:
Key mapping should only be used when it's unavoidable (ex. coming from external component, violate toml convensions, etc). However, in this case this is something that we can easily manage. So let's go with the proper name at config level rather than adding an overhead on mapping